mixins/infra-state.mixin.tf

```diff
@@ -0,0 +1,63 @@
+# <-- BEGIN DOC -->
+#
+# This mixin is meant to be placed in a Terraform config outside the organization's infrastructure monorepo in order to:
+#
+# 1. Instantiate an AWS Provider using roles managed by the infrastructure monorepo. Since Cloud Posse's providers.tf pattern
+#    requires an invocation of the `account-map` component’s `iam-roles` submodule, which is not present in a repository
+#    outside of the infrastructure monorepo.
+# 2. Retrieve outputs from a component in the infrastructure monorepo. Since Cloud Posse’s `remote-state` module expects
```

Comment:
```suggestion
# 2. Retrieve outputs from a component in the infrastructure monorepo. This is required because Cloud Posse’s `remote-state` module expects
```